nextcord/member.py

```diff
@@ -656,6 +656,33 @@ async def kick(self, *, reason: Optional[str] = None) -> None:
         Kicks this member. Equivalent to :meth:`Guild.kick`.
         """
         await self.guild.kick(self, reason=reason)
+        
+    async def timeout(
+        self,
+        timeout: Union[datetime.datetime, datetime.timedelta],
+        *,
+        reason: Optional[str] = None,
+    ) -> None:
+        """|coro|
+
+        Times out this member.
+
+        Parameters
+        -----------
+        timeout: Optional[Union[:class:`~datetime.datetime`, :class:`~datetime.timedelta`]]
+            The time until the member should not be timed out.
+            Set this to None to disable their timeout.
+        reason: Optional[:class:`str`]
+            The reason for editing this member. Shows up on the audit log.
+
+        .. note::
+
+            This is a more direct method of timing out a member.
+            You can also time out members using :meth:`Member.edit`.
+
+        .. versionadded:: 2.0
```

Comment:
this seems to render odd on the docs 
```suggestion
        """|coro|

        Times out this member.

        .. note::

            This is a more direct method of timing out a member.
            You can also time out members using :meth:`Member.edit`.

        .. versionadded:: 2.0

        Parameters
        -----------
        timeout: Optional[Union[:class:`~datetime.datetime`, :class:`~datetime.timedelta`]]
            The time until the member should not be timed out.
            Set this to None to disable their timeout.
        reason: Optional[:class:`str`]
            The reason for editing this member. Shows up on the audit log.
```